code/modules/surgery/prosthetic_replacement.dm

```diff
@@ -27,6 +27,9 @@
 		tool_body_zone = RP.body_zone
 	else if(istype(tool, /obj/item/bodypart))
 		var/obj/item/bodypart/L = tool
+		if(L.brute_dam + L.burn_dam >= L.max_damage/2)
```

Comment:
bodyparts have a get_damage() proc so you don't have to manually add each damage type.